doc/user-guide/genhelp.py

```diff
@@ -0,0 +1,165 @@
+import re
```

Comment:
Spaces are what PEP 8 recommends https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces - and they just feel right for me when dealing with python code. Only happens with python, oddly enough.

Of course I won't use them for the C files.

Hashbang, welp.